src/families/hedera/js-estimateMaxSpendable.ts

```diff
@@ -0,0 +1,28 @@
+import BigNumber from "bignumber.js";
+import type { Account, AccountLike } from "../../types";
+import type { Transaction } from "./types";
+
+export default function estimateMaxSpendable({
```

Comment:
This code is very similar to the `calculateAmount` function in `utils.ts`, can you maybe use it there?

Reply:
👍🏻